aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -1263,6 +1266,7 @@ func resourceAwsKinesisFirehoseDeliveryStream() *schema.Resource {
 
 						"s3_backup_mode": {
 							Type:     schema.TypeString,
+							ForceNew: true,
```

Comment:
Please revert this change. We currently support updates for `splunk_configuration.0.s3_backup_mode` via [`SplunkDestinationUpdate`](https://docs.aws.amazon.com/firehose/latest/APIReference/API_SplunkDestinationUpdate.html):
* Code: https://github.com/terraform-providers/terraform-provider-aws/blob/7f93f3cc123490e8c2ceaee725921a8ab4da8cd7/aws/resource_aws_kinesis_firehose_delivery_stream.go#L1949
* Acceptance testing: This test currently updates `s3_backup_mode` to match the default (`FailedEventsOnly`) but could be updated on lines 645 and 1810 to update to `AllEvents` which would properly test this update https://github.com/terraform-providers/terraform-provider-aws/blob/7f93f3cc123490e8c2ceaee725921a8ab4da8cd7/aws/resource_aws_kinesis_firehose_delivery_stream_test.go#L626

Reply:
Done